### ISSUE TITLE
Discuss and reason the license choice in an issue

### ISSUE THREAD

**Issue by yhan178** [UBC-MDS/wordwright#68]

**Comment by alexzhang0825** [comment]
I am ok with our current MIT license. Mostly because it is simple and ubiquitous. This project is really simple and likely will not attract much popularity among other developers, so we do not need an overcomplicated license covering every possible aspect of fair use. Should the very rare occasion arise, that person would be able to quickly take the package and use it for their own purpose without needing to communicate with any one of us due to how familiar most developers are with the MIT license.

**Comment by yhan178** [comment]
I agree. We are currently using the MIT License. For our project, MIT License is a reasonable choice as it is simple and permissive, making it one of the most popular open-source licenses. It allows other people to freely use, modify, distribute or sell our software, but still require them to credit you and include the original license. These features foster collaboration, especially in the education scenario. Contributors and users can participate effortlessly without getting involved in complicated legal issues.

**Comment by carrieyanyi** [comment]
I agree with using the MIT license. MIT license mostly applies to python packages and our package functions cover uncomplicated text processing through using python.  Our package could be easily downloaded and used, and we also apperciate any contribution or improvement to our package, which also fits into MIT license environment.

**Comment by jinyz8888** [comment]
Same here. Since our package is open source. The MIT License can foster widespread use and contribution to our project, but if we require stricter controls over how our work is used, or if we want to require changes to be shared, we might consider other licenses such as GPL or LGPL.
